Modules/IO/TIFF/src/itkTIFFImageIO.cxx

```diff
@@ -86,7 +86,7 @@ TIFFImageIO::GetColor(unsigned int index, unsigned short * red, unsigned short *
 
   if (m_TotalColors > 0 && m_ColorRed && m_ColorGreen && m_ColorBlue)
   {
-    index %= m_TotalColors;
+    index %= static_cast<unsigned int>(m_TotalColors);
```

Comment:
This line (your added `static_cast`) looks fine to me, but I'm just wondering why `m_TotalColors` wasn't declared `unsigned`. Do you know?

Reply:
m_TotalColors is not unsigned because it can be -1 if image on disk is not palette image. That's why you will see these `m_TotalColors > 0` or `m_TotalColors >-1` everywhere. It is not so clear of a name, but I was thinking doing a different PR for that.